python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/vlan-interfaces.j2

```diff
@@ -315,6 +315,65 @@ interface {{ vlan_interface.name }}
+{%             else %}
+{%                 if vlan_interface.isis_authentication_mode_sha.level_1.key_id is arista.avd.defined %}
+{%                     if vlan_interface.isis_authentication_mode_sha.level_1.rx_disabled is arista.avd.defined %}
+{%                         set auth_cli = auth_cli ~ " rx-disabled" %}
+{%                     endif %}
+{%                     set auth_cli = auth_cli ~ " key-id " ~ vlan_interface.isis_authentication_mode_sha.level_1.key_id ~ " level-1" %}
+   {{ auth_cli }}
+{%                 endif %}
+{%                 if vlan_interface.isis_authentication_mode_sha.level_2.key_id is arista.avd.defined %}
+{%                     if vlan_interface.isis_authentication_mode_sha.level_2.rx_disabled is arista.avd.defined %}
+{%                         set auth_cli = auth_cli ~ " rx-disabled" %}
+{%                     endif %}
+{%                     set auth_cli = auth_cli ~ " key-id " ~ vlan_interface.isis_authentication_mode_sha.level_2.key_id ~ " level-2" %}
+   {{ auth_cli }}
+{%                 endif %}
+{%             endif %}
+{%         endif %}
+{%         if vlan_interface.isis_authentication_mode == "shared-secret" and vlan_interface.isis_authentication_mode_shared_secret is arista.avd.defined %}
+{%             if vlan_interface.isis_authentication_mode_shared_secret.both.profile is arista.avd.defined and vlan_interface.isis_authentication_mode_shared_secret.both.algorithm is arista.avd.defined %}
+{%                 set auth_cli = auth_cli ~ " profile " ~  vlan_interface.isis_authentication_mode_shared_secret.both.profile ~ " algorithm " ~ vlan_interface.isis_authentication_mode_shared_secret.both.algorithm %}
+{%                 if vlan_interface.isis_authentication_mode_shared_secret.both.rx_disabled is arista.avd.defined %}
+{%                     set auth_cli = auth_cli ~ " rx-disabled " %}
+{%                 endif %}
+   {{ auth_cli }}
+{%             else %}
+{%                 if vlan_interface.isis_authentication_mode_shared_secret.level_1.profile is arista.avd.defined and vlan_interface.isis_authentication_mode_shared_secret.level_1.algorithm is arista.avd.defined %}
+{%                     set auth_cli_level1 = auth_cli ~  " profile " ~ vlan_interface.isis_authentication_mode_shared_secret.level_1.profile ~ " algorithm " ~ vlan_interface.isis_authentication_mode_shared_secret.level_1.algorithm %}
+{%                     if vlan_interface.isis_authentication_mode_shared_secret.level_1.rx_disabled is arista.avd.defined %}
+{%                         set auth_cli_level1 = auth_cli_level1 ~ " rx-disabled" %}
+{%                     endif %}
+{%                     set auth_cli_level1 = auth_cli_level1 ~ " level-1" %}
+   {{ auth_cli_level1 }}
+{%                 endif %}
+{%                 if vlan_interface.isis_authentication_mode_shared_secret.level_2.profile is arista.avd.defined and vlan_interface.isis_authentication_mode_shared_secret.level_2.algorithm is arista.avd.defined %}
```

Comment:
this should be `elif`

Reply:
This shouldn't be in elif as we can configure both level1 and level2. Both can be configured at the same time.

s1-leaf1(config-if-Vl100)#isis authentication mode sha key-id 2000 level-1
s1-leaf1(config-if-Vl100)#isis authentication mode sha key-id 2001 level-2
s1-leaf1(config-if-Vl100)#sh active 
interface Vlan100
   isis authentication mode sha key-id 2000 level-1
   isis authentication mode sha key-id 2001 level-2
s1-leaf1(config-if-Vl100)#